data/grc/Arnott_1996_Athenaeus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+# Athenaeus: Arnott 1996
+# Epitome derived from A, contamination from a sibling of A
+
+digraph {
+
+    omega -> A;
+    omega -> sibling_of_A;
+    A -> epitome;
+    sibling_of_A -> epitome [style="dashed"]; # hypothetical ms. derived from the archetype, by definition
+    epitome -> gamma;
+    gamma -> delta;
+    delta -> C;
+    delta -> E;
+    gamma -> Eustathius_ms;
```

Comment:
`gamma -> Eustathius_ms[color="grey"];` (uncertainty)

Reply:
Absolutely! I had missed your example to express the uncertainty of the edge, not of the node.